en/docs/guides/handling-errors.md

```
@@ -141,23 +141,23 @@ To try out storing errors in the store, follow the steps below:
               isAutoCommit: false
         ```
 
-2. Download the [productions.csv file](https://github.com/wso2/docs-ei/blob/master/en/streaming-integrator/docs/examples/resources/productions.csv) and save it in a location of your choice in your machine.
+2. Download the [productions.csv file]({{base_path}}/examples/resources/productions.csv) and save it in a location of your choice in your machine.
 
 3. [Create a Siddhi application](../develop/creating-a-Siddhi-Application.md) as follows and deploy it in the WSO2 Integrator: SI server.
 
     ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)